discovery_engine_core/ai/ai/src/ranker/context.rs

```diff
@@ -129,8 +198,6 @@ pub(super) fn compute_score_for_docs(
     documents
         .iter()
         .map(|document| {
-            // `compute_score_for_embedding` cannot panic because we calculate
-            // the relevances for all positive cois before
```

Comment:
❓ did you remove this comment because it's no longer relevant? (I don't really understand it / why it's there in the first place)

Reply:
correct, this comment is outdated, i guess i forgot to remove it when i refactored the relevance maps. it doesn't apply anymore because the relevances are computed on the fly whithin the mentioned function call now.